algorithms/src/main/kotlin/io/kinference/algorithms/gec/tokenizer/AutoTokenizer.kt

```diff
@@ -0,0 +1,18 @@
+package io.kinference.algorithms.gec.tokenizer
+
+import java.nio.file.Paths
+
+/**
+ * AutoTokenizer class helper for loading Tokenizers
+ */
+class AutoTokenizer {
+    companion object {
+        fun fromPretrained(modelNameOrPath: String): PreTrainedTokenizer {
+            if (modelNameOrPath == "bert-base-uncased") {
+                return BertTokenizer(vocabPath = Paths.get("/Users/Ivan.Dolgov/ivandolgov/projects/vocabs/bert-base-uncased"))
+            } else {
+                throw NotImplementedError("Not implemented yet")
```

Comment:
Let's just add at the top of the function `require` that tells that there is only one model available right now.